examples/rspec-hive.yml.example

```diff
@@ -3,4 +3,10 @@ hive:
   port: 10000
   host_shared_directory_path: /Users/Shared/tmp/spec-tmp-files
   docker_shared_directory_path: /tmp/spec-tmp-files
-  hive_version: 10
\ No newline at end of file
+  hive_version: 13
+  hive_options:
+    'hive.exec.dynamic.partition': 'true'
+    'hive.exec.dynamic.partition.mode': 'nonstrict'
+    'hive.exec.max.dynamic.partitions.pernodexi': '100000'
```

Comment:
I don't think this setting exist and I think it has never existed in Hive, it should be `hive.exec.max.dynamic.partitions.pernode`